src/components/Page/Base.js

```diff
@@ -11,6 +12,7 @@ import config from "@/promisetracker/config";
 /**
  * Base page that can be used to build all other pages.
  */
+const GTM_ID = process.env.GTM_ID || "GTM-P8QSVKT";
```

Comment:
Ola @kilemensi  Concerning this part
`const GTM_ID = process.env.GTM_ID`
The `GTM_ID` seems to only be available in the server side. I had to create something like `NEXT_PUBLIC_GTM_ID=` in `.env.local` for me to get the `GTM_ID` show in the client. Is this the correct approach? The other `env` seems to be used in the server.

Reply:
Heita... Yes, if you want Next.js to expose an env var to the client, it must **start** with `NEXT_PUBLIC_`.

Should this var be a secret or it's OK for it to be visible to the clients @esirK ?

Reply:
No, it's no secret since it will still be available on the website after deploying.